### PR TITLE
dashboards: overview: qps and latency w/ cortex_request_duration_seconds

### PR DESCRIPTION
**Pull request by krajorama**
#### What this PR does

Allow using  cortex_request_duration_seconds native histogram in
overview dashboard everywhere.

#### Which issue(s) this PR fixes or relates to

Followup to #7627
Related to #7154 
Depends on https://github.com/grafana/jsonnet-libs/pull/1150

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: operations/mimir-mixin/jsonnetfile.json
Comment (johannaratliff) [inline operations/mimir-mixin/jsonnetfile.json:1]: Do we want to lock this in not on master branch on the mixin jsonnetfile or is this an oversight?

**Review by johannaratliff — COMMENTED** [review]
The logic on the dashboards and your example dashboard shown to me looks great. Just want to check about the versioning on jsonnetfile.json and then I'm good to approve

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,8 +49,8 @@
   * `MimirIngesterFailsEnforceStrongConsistencyOnReadPath`
 * [ENHANCEMENT] Dashboards: add in-flight queries scaling metric panel for ruler-querier. #7749
 * [ENHANCEMENT] Dashboards: renamed rows in the "Remote ruler reads" and "Remote ruler reads resources" dashboards to match the actual component names. #7750
-* [ENHANCEMENT] Dashboards: allow switching between using classic of native histograms in dashboards. #7627
-  * Overview dashboard, Status panel, `cortex_request_duration_seconds` metric.
+* [ENHANCEMENT] Dashboards: allow switching between using classic of native histograms in dashboards. #7627 #7674
```
Comment (johannaratliff) [inline CHANGELOG.md:52]: `allow switching between use of classic or native histograms in dashboards.`

File: operations/helm/charts/mimir-distributed/CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [ENHANCEMENT] Dashboards: allow switching between using classic of native histograms in dashboards. #7627 #7674
```
Comment (johannaratliff) [inline operations/helm/charts/mimir-distributed/CHANGELOG.md:32]: `allow switching between use of classic or native histograms in dashboards.`
or 
`allow switching between using classic or native histograms in dashboards.`

File: operations/mimir-mixin/jsonnetfile.json
Comment (krajorama) [reply operations/mimir-mixin/jsonnetfile.json:1]: nono, I was just waiting to merge the jsonnet libs PR

**Comment by dimitarvdimitrov** [comment]
The CHANGELOG has just been cut to prepare for the next Mimir release. Please rebase `main` and eventually move the CHANGELOG entry added / updated in this PR to the top of the CHANGELOG document. Thanks!

**Review by pstibrany — APPROVED** [review]

**Pull request merged**